app/controllers/errors_controller.rb

```diff
@@ -1,4 +1,4 @@
-class ErrorsController < ApplicationController
+class ErrorsController < PublicPagesController
```

Comment:
I think this one should probably inherit from `ActionController::Base` since its an errors controller we don't think we would ever want any of the hooks, or other changes from ApplicationController running

Reply:
it looks like this one needs to be PublicPagesController, because we currently still show the shared header on the errors page, and this relies on having `current_admin` and `current_user` from `ApplicationController`.